static/app/views/performance/newTraceDetails/traceDrawer/details/styles.tsx

```diff
@@ -525,9 +529,59 @@ function HighLightsOpsBreakdown({event}: {event: EventTransaction}) {
   );
 }
 
+function HighLightEAPOpsBreakdown({node}: {node: TraceTreeNode<TraceTree.EAPSpan>}) {
+  const theme = useTheme();
+  const breakdown = node.eapSpanOpsBreakdown;
+
+  if (breakdown.length === 0) {
+    return null;
+  }
+
+  breakdown.sort((a, b) => b.count - a.count);
```

Comment:
`.sort()` mutates the original array from the `node` prop. You probably want to use `toSorted` which returns a sorted copy, or you can copy `breakdown` yourself and sort the copy.

Reply:
Agree @edwardgou-sentry , pushed the change